content/ros/README.md

```diff
@@ -1,6 +1,6 @@
 # ROS 
-ROS stands for Robot Operating System, it is a collection of libraries and tools which you can use for developing software applications for robots. These libraries and tools that comprises ROS are installed on top of Ubuntu, they are released as distributions. We will be using **ROS Kinetic** distribution which has a long-term support (LTS) until April, 2021.
+ROS stands for Robot Operating System, it is a collection of libraries and tools which you can use for developing software applications for robots. These libraries and tools that comprises ROS are installed on top of Ubuntu, they are released as distributions. We will be using **ROS Noetic** distribution which has a long-term support (LTS) until May, 2025.
 
-:warning:  Please make sure you have installed Ubuntu 16 not the latest version of Ubuntu. As ROS Kinetic is only compatible with Ubuntu 16.
+:warning:  Please make sure you have installed Ubuntu 20, as ROS Noetic is compatible with Ubuntu 20.
 
-In this folder, there are installation instructions to setup ROS, and the material that will be covered during ROS sessions of the foundation course.
+However, up to date the ROS **Kinetic** was used and in this folder, there are installation instructions to setup ROS **Kinetic** in case anyone would be interested.
```

Comment:
Please remove this line; no one should try to set up Ubuntu 16.04/ROS kinetic anymore.